spring-boot-starter-rocketmq/README.md

```diff
@@ -0,0 +1,207 @@
+# spring-boot-starter-rocketmq
+
+[中文](./README_zh_CN.md)
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
+
+Help developers quickly integrate [RocketMQ](http://rocketmq.apache.org/) in [Spring Boot](http://projects.spring.io/spring-boot/). Support the Spring Message specification to facilitate developers to quickly switch from other MQ to RocketMQ.
+
+
+Features:
+
+- [x] synchronous transmission
+- [x] synchronous ordered transmission
+- [x] asynchronous transmission
+- [x] asynchronous ordered transmission
+- [x] orderly consume
+- [x] concurrently consume(broadcasting/clustering)
+- [x] One-way transmission
+- [ ] transaction transmission
+- [ ] Pull consume 
+
+## Quick Start
+
+```xml
+<!--add dependency in pom.xml-->
+<dependency>
+    <groupId>org.apache.rocketmq</groupId>
+    <artifactId>spring-boot-starter-rocketmq</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+</dependency>
+```
+
+> Info:
+>
+> if this version not exists in maven central repository, you can use `mvn clean install` build it by self. If you want use a release version, please see another project: [https://github.com/QianmiOpen/spring-boot-starter-rocketmq](https://github.com/QianmiOpen/spring-boot-starter-rocketmq)
```

Comment:
Hi, please remove this line, we will have an apache release version~

Reply:
Ok, will be remove.